backend/btrixcloud/basecrawls.py

```diff
@@ -688,6 +733,30 @@ async def update_crawl(
     @app.post("/orgs/{oid}/all-crawls/delete", tags=["all-crawls"])
     async def delete_crawls_all_types(
         delete_list: DeleteCrawlList,
+        user: User = Depends(user_dep),
         org: Organization = Depends(org_crawl_dep),
     ):
-        return await ops.delete_crawls_all_types(delete_list, org)
+        crawls: list[str] = []
```

Comment:
Can this logic be moved into the ops delete_crawls_all_types() also, so can call it with one list, user and org?